xcdat/bounds.py

```diff
@@ -374,26 +427,12 @@ def _create_bounds(self, axis: CFAxisKey, coord_var: xr.DataArray) -> xr.DataArr
         diffs = np.insert(diffs, 0, diffs[0])
         diffs = np.append(diffs, diffs[-1])
 
-        # `cftime` objects only support arithmetic using `timedelta` objects, so
-        # the values of  `diffs` must be casted from `dtype="timedelta64[ns]"`
-        # to `timedelta` objects.
-        if axis == "T" and issubclass(type(coord_var.values[0]), cftime.datetime):
-            bounds = create_time_bounds(coord_var)
-            return bounds
```

Comment:
`if axis == "T"` is not necessary in `_create_bounds()`, and was moved to `add_bounds()`/`add_missing_bounds()`.

Reply:
Also we should be supporting time coordinates as `datetime` objects (removed `issubclass(...)` conditional)